Decisions.Docusign/DocusignSettings.cs

```diff
@@ -99,15 +101,5 @@ public void Initialize()
             // Create default settings object
             ModuleSettingsAccessor<DocusignSettings>.GetSettings();
         }
-
-        public void SetDocusignSettings(DocusignSettings settings)
-        {
-            settings.Store();
-        }
-        
-        public DocusignSettings GetDocusignSettings()
-        {
-            return DSServiceClientFactory.DsSettings;
-        }

```

Comment:
Were these 2 methods entirely unused?

Reply:
Yes, their usages were refactored out or moved with the last cleanup commit for [DT-036344] Docusign - Signatures try to start on document zero.